### ISSUE TITLE
Ingress failing when RBAC enabled on edge

### ISSUE THREAD

**Issue by moredhel** [ubuntu/microk8s#821]
Hi, I'm in the process of installing microk8s, it seems that the Role is misconfigured for the ingress controller on the `--edge` release.

I had to add the the following to get it working:

```
- apiGroups:
  - ""
  resources:
  - services
  - ingresses
  verbs:
  - get
  - list
```

Attached is the version info, installed using:

`snap install microk8s --edge --classic`

[inspection-report-20191122_234153.tar.gz](https://github.com/ubuntu/microk8s/files/3881715/inspection-report-20191122_234153.tar.gz)

Thanks for the awesome project!

**Comment by balchua** [comment]
@moredhel I've been looking at the rbac, it does have `ingress` included.
https://github.com/ubuntu/microk8s/blob/797720e2d1e74030fc3d8df5d291469c6082aaac/microk8s-resources/actions/ingress.yaml#L47-L52

May i know what error you are getting?  I also couldn't find any error reported by the apiserver.

**Comment by moredhel** [comment]
Hi Balchua,

Thanks for the response.

I did get it working, (as mentioned above), but that was not present in the role definition that I was running with.

I've since patched it on my local instance, so don't have access to the logs.

The error was Nginx complaining about creating a ConfigMap for leader election.

The following issue describes what I experienced: https://github.com/kubernetes/ingress-nginx/issues/798

Thanks
